### PR TITLE
Fixed installer collation selection

### PR DESCRIPTION
**Pull request by MekDrop**
Same thing that I found when I was changing installer to make work it with Aura.SQL for #527 pull request.

Because of missing [prototype.js](http://prototypejs.org) in installer user can't select collation for database based on selected encoding. So, missing JS added.

### CODE REVIEW

**Comment by MekDrop** [comment]
@fiammybe that is possible, but I was just lazy to find out how to replace Ajax.Updater method with same from jQuery (I don't know much how prototype.js works). This pull request just makes this work and not anything improves.

**Comment by fiammybe** [comment]
ok, I'll test it in this form. If I can test the upgrade procedure from 1.2 and 1.3 we can go with this in final release and clean up in a following one. After all, it's just the installer (only used once normally).

**Comment by skenow** [comment]
It does work with the addition and inclusion of prototype.js, but I'd rather not see this regression.

File: htdocs/install/page_dbsettings.php [hunk 1]
```diff
@@ -347,9 +347,9 @@ function xoFormFieldCharset($name, $value, $label, $help = '', $link) {
 	<script type="text/javascript">
 		function setFormFieldCollation(id, val) {
 			if (val == '') {
-				$(id).style.display='display';
```
Comment (fiammybe) [inline htdocs/install/page_dbsettings.php]: If I understand it correctly, this class should not be 'display', but it should be 'none'?

Comment (MekDrop) [reply htdocs/install/page_dbsettings.php]: Yes, this was fixed. I think you are looking into old version of code.

**Pull request merged**